config/casestudies/hdi.yml

```diff
@@ -0,0 +1,163 @@
+                      "type": "integer"
+                    },
+                    "currently_occupied_rooms": {
+                      "type": "integer"
+                    }
+                  },
+                  "required": [
+                    "id",
+                    "name"
+                  ]
+                }
+              },
+              "required": [
+                "eventType",
+                "hotel"
+              ]
+            }
+      ```
+  codegen: |
+    Our team does not use code generation tools. 
+    However, our customers are free to use any tool they prefer to generate DTOs from the AsyncAPI document.
+schemas: #It is useful and interesting to learn how teams handle messages schemas.
+  description: JSON Schema
+  storage: A Git repository functions as a self-service portal where customers can manage their own configurations for the Azure Service Bus. 
+    This includes defining their own message schemas for any topics they own.
+  registry: none
+  versioning: The customer has the freedom to choose the versioning for their own message-schema files.
+  validation: Validation using [Ajv](https://ajv.js.org/), is used solely for the purpose of validating that the JSON data adheres to the expected schema. 
+    It does not perform any other form of validation or processing of the data.
+asyncapi: #More specific details about AsyncAPI itself and how is it used.
+  usecase: |
+    - The AsyncAPI file is used for documentation purposes by the platform team. 
+      It provides a comprehensive overview of the asynchronous communication in our system, including the available topics and the structure of the messages.
+    - The AsyncAPI document serves as a source for generating data transfer objects (DTOs) in Java, for example, by using the AsyncAPI Java generator. 
+      These DTOs are used in the code to ensure type safety and to provide a clear understanding of the data structure.
+  versions:
+    - '3.0.0'
+  storage: |
+    A seperate Git repository functions as a "customer Wiki", serving as a central location for all relevant documents about the existing infrastructure. 
+    This includes the AsyncAPI document, which provides a comprehensive overview of the asynchronous communication in our system.
+  editing: |
+    If changes for any topic are applied by a customer, the documentation pipeline is triggered. 
+    This pipeline uses a bash script to read the updated configuration from the customer's repository. 
+    It then generates a new AsyncAPI file reflecting these changes.
+  maintainers: | 
+    No specific maintainers are assigned to the AsyncAPI document. The document is automatically generated by a pipeline.
+  audience: #Specify if AsyncAPI audience is internal or external or both.
+    internal: true
+    external: false
+  extensions: |
+    none
+  documentation: |
+    Documentation is generated via AsyncAPI CLI and published to the wiki right after the AsyncAPI file is generated.
```

Comment:
when you write wiki, 

the central repo with all AsyncAPI docs and generated markdown - do you use GitHub Wiki solution to render these markdowns? or? why you call it wiki?